contracts/Example3.sol

```diff
@@ -1,14 +1,19 @@
 pragma solidity ^0.8.4;
 
 contract Example3 {
-    uint256 marginPercentage = 30;
+    // uint256 marginPercentage;
+
+    // constructor() {
+    //   marginPercentage = 30;
+    // }
+
     function getOwnerMargin(uint256 amount)
-        public
-        view
+        external
+        pure
         returns (uint256 amountForSender, uint256 amountForOwner)
     {
-        amountForSender = (amount * (100 - marginPercentage)) / 100;
-        amountForOwner = (amount * marginPercentage) / 100;
+        amountForSender = (amount * 70) / 100;
```

Comment:
It is better to have `marginPercentage` stored in a variable since this is being used twice (indirectly or directly). You can move `uint256 marginPercentage = 30;` to inside function to avoid accessing storage twice